docs/integrations/llms/openai.md

```diff
@@ -134,3 +140,81 @@ Shows up like this in Logfire:
   ![Logfire OpenAI Streaming](../../images/logfire-screenshot-openai-stream.png){ width="500" }
   <figcaption>OpenAI streaming response</figcaption>
 </figure>
+
+## OpenAI Agents
+
+We also support instrumenting the [OpenAI "agents"](https://github.com/openai/openai-agents-python) framework.
+
+```python hl_lines="5"
+import logfire
+from agents import Agent, Runner
+
+logfire.configure()
+logfire.instrument_openai_agents()
+
+agent = Agent(name="Assistant", instructions="You are a helpful assistant")
+
+result = Runner.run_sync(agent, "Write a haiku about recursion in programming.")
+print(result.final_output)
+```
+
+_For more information, see the [`instrument_openai_agents()` API reference][logfire.Logfire.instrument_openai_agents]._
```

Comment:
@samuelcolvin this link feels redundant, there's nothing to see there.

Reply:
I think having a links to this reference is good.